test/data/pipeline.update.yaml

```diff
@@ -3,6 +3,7 @@ checkoutUrl: git@github.com:screwdriver-cd/data-schema.git#branch
 rootDir: src/app/test
 autoKeysGeneration: false
 settings:
+    aliasName: scwdvr-cd
```

Comment:
```suggestion
```

Reply:
I didn;t get it why we want remove `aliasName` from here?

Reply:
oops misread
ignore this comment